src/panels.h

```diff
@@ -8,6 +8,8 @@
 #include <string>
 #include <vector>
 
+#include "character.h"
```

Comment:
Instead of including this header, you should be able to just declare `class character;` instead.